lib/nuget-parser/index.ts

```diff
@@ -92,3 +107,10 @@ export async function buildDepTreeFromFiles(
     targetFramework,
     packagesFolder);
 }
+
+export async function getTargetFrameworkFromPackagesConfig(fileContent: string): Promise<TargetFramework | undefined> {
+  const extractedFrameworks = await depsParser.extractTargetFrameworksFromProjectConfig(fileContent);
```

Comment:
Sort the returned list, and pick the first element, to be more deterministic, as the order in the list might change as packages get added.

Reply:
Changed